python/llm/src/ipex_llm/transformers/npu_model.py

```diff
@@ -423,23 +423,32 @@ def load_low_bit(cls, pretrained_model_name_or_path: str, *model_args, **kwargs)
 
         if enable_cpp_backend:
             from .npu_models.npu_llm_cpp import load_model_from_file
-            from .npu_models.convert import generate
-            dummy_model = torch.nn.Module()
+            from .npu_models.convert import generate, general_convert, prepare_input_ids, causal_lm_forward
+            config = AutoConfig.from_pretrained(os.path.join(pretrained_model_name_or_path, "config.json"),
+                                                trust_remote_code=trust_remote_code)
+            with torch.device('meta'):
+                model = transformers.AutoModelForCausalLM.from_config(config, trust_remote_code=trust_remote_code)
             try:
                 model_ptr = load_model_from_file(pretrained_model_name_or_path)
-                dummy_model.config = PretrainedConfig.from_dict(config_dict)
-                dummy_model.model_ptr = model_ptr
-                dummy_model.save_directory = pretrained_model_name_or_path
-                dummy_model.kv_len = config_dict['kv_len']
-                dummy_model.vocab_size = config_dict['vocab_size']
+                model.config = PretrainedConfig.from_dict(config_dict)
```

Comment:
Cool, upated.

Reply:
Probably not, but I see it may be used here: https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/src/ipex_llm/transformers/npu_models/convert.py#L364
Maybe keep it for now.